pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -116,12 +124,23 @@ func newRepositoryWithClient(registryClient client.Interface, quotaClient kclien
 		namespace:      nameParts[0],
 		name:           nameParts[1],
 		pullthrough:    pullthrough,
+		acceptschema2:  acceptschema2,
 		cachedLayers:   cachedLayers,
 	}, nil
 }
 
+func getBoolOption(name string, defval bool, options map[string]interface{}) bool {
+	if value, ok := options[name]; ok {
+		if b, ok := value.(bool); ok {
+			return b
+		}
+	}
+	return defval
+}
+
 // Manifests returns r, which implements distribution.ManifestService.
 func (r *repository) Manifests(ctx context.Context, options ...distribution.ManifestServiceOption) (distribution.ManifestService, error) {
+	context.GetLogger(r.ctx).Debugf("(repository).Manifests: starting")
```

Comment:
Do we care about those logs, or these are just leftovers from your debugging? Same for the next one.